nixos/modules/services/web-servers/caddy/default.nix

```diff
@@ -245,15 +245,22 @@ in
     };
 
     acmeCA = mkOption {
-      default = "https://acme-v02.api.letsencrypt.org/directory";
+      default = null;
       example = "https://acme-staging-v02.api.letsencrypt.org/directory";
```

Comment:
We should adapt the example

Reply:
To what?

Reply:
I'm guessing maybe the testing/staging endpoint for zerossl but IDK if that actually exists

https://caddyserver.com/docs/caddyfile/options#acme-ca

> It is strongly recommended to set this to Let's Encrypt's [staging endpoint](https://letsencrypt.org/docs/staging-environment/) for testing or development. Default: ZeroSSL and Let's Encrypt's production endpoints.

Reply:
@SuperSandro2000 can you please share what you had in mind?

Reply:
We should change the example/description to mention those two endpoints, otherwise we loose information here.